setup/bin/roslin_request_to_yaml.py

```diff
@@ -214,6 +218,18 @@ def calculate_abra_ram_size(grouping_dict):
     parser.add_argument("--pipeline-name-version",action="store",dest="pipeline_name_version",help="Pipeline name/version (e.g. variant/2.5.0)",required=True)
     parser.add_argument("--clinical", help="the clinical data file", required=False)
     args = parser.parse_args()
+    # hacky way of grabbing clinical data for using pdx ref
+    pdx_genome = False
+    clinical_file_glob = glob.glob(os.path.join(os.path.dirname(args.grouping), '*sample_data_clinical.txt'))
```

Comment:
Any reason we are not using the --clinical input parameter for this?

Reply:
I missed that `--clinical` is added. It wasn't there in the [code](https://github.com/mskcc/roslin-variant/commit/4794db96530a62ae37e00c5f789ed74a9c54ecc0) before. And I wasn't sure will it break something if I change it. Should I change to use only `--clinical`?